LondonBritishLibrary/orient/BLorient2198.xml

```diff
@@ -30,27 +30,136 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <msIdentifier>
                         <repository ref="INS00001BL"/>
                         <idno>BL Oriental 2198</idno>
+                        <altIdentifier>
+                            <idno>Strelcyn 15</idno>
+                        </altIdentifier>
                     </msIdentifier>
                     
                     <msContents>
                         <summary/>
                         <msItem xml:id="ms_i1">
                             <locus from="1r" to="13r"/>
                             <title type="incomplete" ref="LIT3585Taamme"/>
-                            <note>The beginning of the text is missing.</note>
-                            <textLang mainLang="gez"></textLang>
+                            <note>Numbered from second to seventh miracle. The miracle numbered first and the beginning of the second miracle is 
+                                missing.</note>
+                                <msItem xml:id="ms_i1.1">
+                                    <locus from="1r" to="3r"/>
+                                    <title type="incomplete">Second miracle</title>
```

Comment:
LIT3586Miracle

we have an ID for all Miracles, with bibliography and with incipits, if you know the incipit or pages in Budge and/or Cerulli it is very easy to define them

Reply:
It would be best to put what you have in the note in the title, and the title in the note
```
<title type="incomplete" ref="LIT3586Miracle">How the Virgin Mary accepted the Book of her Miracles compiled by St Hildephonsus, Bishop of Toledo</title>
<note>numbered as Miracle 2</note>
```
you do not need the bibliography since it belongs to the work record (you can check if it is there but for MM it surely is)

Reply:
same for others